### DISCUSSION TITLE
Welcome to Capstone-Project Discussions!

### DISCUSSION THREAD

**Discussion by Carlie16604** [team-black-market/Capstone-Project#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Carlie16604** [comment]
How to implement the backend API?

**Reply by AuztinC** [reply]
not sure, ask prof
